openshift/nginx.yml.j2

```
@@ -67,9 +72,24 @@ data:
                 proxy_method     GET;
              }
 
+             location = /flower-metrics {
+               proxy_pass http://flower:5555/metrics;
+               proxy_method     GET;
+             }
+
              location / {
```

```suggestion
             location /flower {
```

I think this should be rather something like this, in order to be explicit about where this route is going to lead :)

---

And keep returning `404` on `/`.

---

I tried really hard to do that, without any success.
I was not able to serve the static content as expected by flower and at the end I give up... if you think this is necessary we can try to configure it together.

---

After playing with this a little bit, I don't think this can be done, or at least I wasn't able to figure it out either. Flower needs some more resources (CSS, JS) loaded to run, and loading these resources will happen using the hostname, but in order to work, it should use `hostname/flower`. And I wasn't able to figure out how to configure this in flower.

So let's keep everything as it is.